src/ast/mod.rs

```diff
@@ -7766,6 +7766,27 @@ impl fmt::Display for RenameTable {
     }
 }
 
+/// table object for insertion
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub enum TableObject {
+    // for simple table name
```

Comment:
```suggestion
    /// Table specified by name.
    /// Example:
    /// ```sql
    /// INSERT INTO my_table
    /// ```
```